client/my-sites/importer/importing-pane.jsx

```diff
@@ -254,11 +261,21 @@ class ImportingPane extends React.PureComponent {
 	}
 }
 
-const mapDispatchToProps = dispatch => ( {
-	mapAuthorFor: importerId => ( source, target ) =>
-		setTimeout( () => {
-			dispatch( mapAuthor( importerId, source, target ) );
-		}, 0 ),
-} );
+const mapDispatchToProps = dispatch => {
+	const boundActionCreators = bindActionCreators(
+		{
+			startImporting,
+		},
+		dispatch
+	);
+
+	return {
+		...boundActionCreators,
+		mapAuthorFor: importerId => ( source, target ) =>
+			defer( () => {
+				dispatch( mapAuthor( importerId, source, target ) );
+			} ),
+	};
+};
```

Comment:
A coupled notes on this here…

I'm curious about our use of `bindActionCreators` because I'm not sure it's adding any value or making the code any clearer. Is there a reason not to return the simpler `startImporting: () => dispatch( startImporting )`?

Second, what's the need for the `defer()`/`setTimeout()` here? I don't think it should be necessary. Without it our function can turn into the shorthand syntax as well…

```js
const mapDispatchToProps = dispatch => ( {
	startImporting: () => dispatch( startImporting() ),
	mapAuthorFor: importerId => ( source, target ) => dispatch( mapAuthor( importerId, source, target ) ),
} )
```

Reply:
For the `defer` - this might have been related to flux, and some weirdness around dispatching multiple actions. This comment would seem to indicate that's the case:
https://github.com/Automattic/wp-calypso/pull/23930#discussion_r179778504

I can look to simplify the `mapDispatchToProps`. One way could be that the `mapAuthorFor` higher order function becomes a method of the class:
```
mapAuthorFor: importerId => ( source, target ) => this.props.mapAuthor( importerId, source, target );
```
then I think I can use the shorthand for mapDispatchToProps and inline it:
```
export default connect( null, {
  startImporting,
  mapAuthor,
} )( localize( ImportingPane ) );
```